pkg/cmd/cli/install/install.go

```diff
@@ -130,9 +134,44 @@ func NewCommand(f client.Factory) *cobra.Command {
 	return c
 }
 
+func GetVeleroVersion(f client.Factory) (string, error) {
+	// Retrieve velero version on server side
+	serverStatusGetter := &serverstatus.DefaultServerStatusGetter{
+		Namespace: f.Namespace(),
+		Timeout:   5 * time.Second,
+	}
+	var veleroClient velerov1client.ServerStatusRequestsGetter
+	client_v, err := f.Client()
+	cmd.CheckError(err)
+	veleroClient = client_v.VeleroV1()
+	serverStatus, err := serverStatusGetter.GetServerStatus(veleroClient)
+	if err != nil {
+		return "", err
+	}
+	version := serverStatus.Status.ServerVersion
+    return version, nil
+}
+
+func CheckVeleroVersion(version string) bool {
+	digits := strings.Split(version, ".")
+	if digits[1] == "0" || digits[1] == "1" || digits[1] == "2" {
+		return false
+	}
+	return true
+}
+
 func (o *InstallOptions) Run(c *cobra.Command, f client.Factory) error {
 	var resources *unstructured.UnstructuredList
 
+	veleroVersion, err := GetVeleroVersion(f)
+	if err != nil {
+		fmt.Errorf("Failed to check velero version.")
+		return err
+	}
+	if !CheckVeleroVersion(veleroVersion) {
+		fmt.Printf("WARNING: Velero version %s is prior to v1.3.0. Velero-plugin-for-vsphere requires velero version to be v1.3.0 or above.\n", veleroVersion)
+	}
+
```

Comment:
It would be better to check it after the local mode check

Reply:
Move after local mode check